src/include/liburing/io_uring.h

```diff
@@ -109,12 +109,20 @@ struct io_uring_sqe {
 
 enum {
 	IOSQE_FIXED_FILE_BIT,
+#define IOSQE_FIXED_FILE_BIT IOSQE_FIXED_FILE_BIT
 	IOSQE_IO_DRAIN_BIT,
+#define IOSQE_IO_DRAIN_BIT IOSQE_IO_DRAIN_BIT
+
```

Comment:
Is this additional blank line intentional?

Reply:
No. I will remove it